jac-uk/admin
# #1755 Remove 'Download as Doc' button

## What's included?
Removes the 'Download as Doc' button. 
Closes #1755 

## Who should test?
✅ Product owner
✅ Developers

## How to test?
The 'Download as Doc' button has been removed from Application view. 
Please check that Admin still loads ok (we have removed packages)
Please check Application view is still working.

## Risk - how likely is this to impact other areas?
🟠 Medium risk - this does change code that is shared with other areas

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
